test/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: "3.9"
```

Comment:
We can use red panda instead of zookeeper-kafka like [that](https://github.com/Trendyol/kafka-konsumer/blob/main/test/integration/docker-compose.yml). You can find more information [in this article](https://levelup.gitconnected.com/boosting-your-kafka-integration-tests-using-redpanda-with-go-247e4276c61d)